client/src/components/Header/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import Menu from '../Menu';
+import Hamburger from '../Hamburger';
+
+const Wrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  padding: 1.5rem 2rem;
+  z-index: 101;
+  position: fixed;
+  top: 0;
+  background: ${p => p.theme.color.darkBlue};
+  opacity: 0.95;
+
+  & + * {
+    transition: filter 0.4s;
+    filter: ${p =>
+      p.isOpen ? 'blur(4px) grayscale(50%) opacity(50%)' : 'none'};
+  }
+`;
+
+const Header = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <Wrapper isOpen={isOpen}>
+      <div>logo</div>
+      <Hamburger isOpen={isOpen} handleToggle={() => setIsOpen(!isOpen)} />
+      <Menu isOpen={isOpen} handleToggle={() => setIsOpen(!isOpen)} />
+    </Wrapper>
+  );
```

Comment:
You should do something like this to ensure the state changes properly every time. You use the existing state to set set the next state. That way it will always toggle as expected.
```suggestion
  const [isOpen, setIsOpen] = useState(false);
 
   function toggleIsOpen() {
      setIsOpen(isOpen => !isOpen);
   }

  return (
    <Wrapper isOpen={isOpen}>
      <div>logo</div>
      <Hamburger isOpen={isOpen} handleToggle={toggleIsOpen} />
      <Menu isOpen={isOpen} handleToggle={toggleIsOpen} />
    </Wrapper>
  );
```